docs/Books.rst

```diff
@@ -1 +1,284 @@
+    - The book’s description can be provided here.
+- Pages
+    - This is the total number of pages.
+- Edited volume?
+    - If this is an edited volume, tick this box. This will ensure the citation is adjusted to match.
+- Open Access?
+    - If this monograph is Open Access, tick this box. This will ensure the monograph is marked as open access and made available on the website.
+- Date embargo
+    - This will specify the date until which the title is embargoed.
+- Date published
+    - This will specify the date of publication (this can be before the upload date for example if the official publication date precedes the date on which the digital edition was made available).
+- Publisher name
+    - This is a required field.
+- Publisher location
+    - This is a required field.
+- Cover
+- DOI
+    - DOIs for monographs and chapters cannot be minted directly through Janeway, but if a DOI has been chosen, it can be added here. To mint a DOI for a monograph or chapter after it has been uploaded, you can do so through the `Crossref website <https://apps.crossref.org/webDeposit/>`_
+- ISBN
+- Purchase URL
+    - If this book has an option for physical purchase, this is where this can be linked. It will show as an option next to the ‘Read’ and ‘Download’ buttons on the book’s page.
+- Remote URL
+    - If this monograph is not hosted on Janeway but you would like it linked on the website, this is where to provide the link. If the monograph is hosted on Janeway (if you are uploading manuscript files), there is no need for this.
+- Remote label
+    - This will set the label for the link above. If not set, it will display the domain name.
+- Licence information
+    - Add copyright and/or licence information here.
+- Custom how-to-cite
+    - To be used only if the citation block generated by Janeway is not suitable.
+
+After the required fields above have been filled in and you have pressed ‘Save Book’, you can now move onto the following steps (displayed on the right-hand side in Janeway):
+
+Contributors
+~~~~~~~~~~~~~
+All contributors to a volume can be entered here. If you are uploading individual chapters in addition to the full manuscript, the respective authors and/or contributors for these chapters will need to be entered here. If they are not entered here, they can not be selected as contributors when uploading individual chapters.
+
+.. figure:: nstatic/books_add_contributor.png
+    :alt: The page for adding new contributors and its fields.
+
+This page contains the following fields:
+
+- First name
+    - This is a required field.
+- Middle name
+- Last name
+    - This is a required field.
+- Affiliation
+    - This is a required field.
+- Email
+- Sequence
+    - This determines the order in which contributors are displayed and will be auto-filled (but can be edited). This is a required field.
+
+Formats
+~~~~~~~~~
+
+This is where manuscript files are uploaded. Various file types can be used, such as PDF, ePub and Mobi.

```

Comment:
W3C refer to them as EPUB rather than ePub or ePUB so perhaps we should follow that convention?

Reply:
Done :)